techniques/failures/F111.html

```diff
@@ -159,22 +159,22 @@ <h2>Related Techniques</h2>
 			</ul>
 		</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
 			<section class="test-procedure" id="procedure">
-				<h3>Procedure</h3>
-				<p>For all controls with a visible label (e.g., link text, button text, programmatically linked label, images in links or buttons with text, etc.), check that:</p>
+				<h3>手順</h3>
+				<p>可視ラベル (リンクテキスト、ボタンテキスト、プログラムによりリンクされたラベル、リンク内又はボタン内にあるテキストを伴う画像など) を持つ全てのコントロールについて:</p>
 				<ol>
-					<li>The control has a visible text label.</li>
-					<li>The control has an accessible name.</li>
-          <li>The accessible name contains the text that appears as the visible label.</li>
+					<li>コントロールは可視のテキストラベルがある。</li>
```

Comment:
「は……ある」というのは変なので、「は」→「に」もしくは「には」でしょうか。


```suggestion
					<li>コントロールに可視のテキストラベルがある。</li>
```